src/providers/twitch/TwitchAccount.cpp

```diff
@@ -264,11 +264,31 @@ void TwitchAccount::loadUserstateEmotes(std::weak_ptr<Channel> weakChannel)
             [this, weakChannel](QJsonArray emoteSetArray) {
                 auto emoteData = this->emotes_.access();
                 auto localEmoteData = this->localEmotes_.access();
-                for (auto emoteSet_ : emoteSetArray)
+
+                std::unordered_set<QString> subscriberChannelIDs;
+                std::vector<IvrEmoteSet> ivrEmoteSets;
```

Comment:
```suggestion
                std::vector<IvrEmoteSet> ivrEmoteSets;
                ivrEmoteSets.reserve(emoteSetArray.size());
```

optimization to avoid expensive resizing